animations/src/main/java/com/chiralbehaviors/CoRE/kernel/Bootstrap.java

```diff
@@ -151,11 +148,8 @@ public void clear() throws SQLException {
     private void constructKernelWorkspace() throws IOException, SQLException {
         Agency core = find(WellKnownAgency.CORE);
         Product kernelWorkspace = find(WellKnownProduct.KERNEL_WORKSPACE);
-        Product workspace = find(WellKnownProduct.WORKSPACE);
-        Relationship isA = find(WellKnownRelationship.IS_A);
 
-        // Ain
-        createKernelWorkspace(core, kernelWorkspace, workspace, isA);
```

Comment:
simplified the proto version of the workspace, pushing much more into the kernel.2.wsp